src/actions/admin.ts

```diff
@@ -379,6 +379,15 @@ export function purgeElasticsearchIndexes(): ActionFunc {
     });
 }
 
+export function purgeBleveIndexes(): ActionFunc {
```

Comment:
does this action need to live in this library? cannot be added to the Webapp actions instead?

Reply:
I added it here because consistency, but probably makes sense to move the whole `admin.ts` to `mattermost-webapp`

Reply:
I cannot agree more with that!

Reply:
For now I going to close this PR and move this changes to the webapp, but I think we can't remove the other actions from here before 6.0.

Reply:
Actually, I still need the changes for the client, so I going to remove the actions and keep the client function.